azure-functions/acbs-function/api.js

```diff
@@ -1,75 +1,126 @@
 /**
- * ACBS Functions API Library deals with following HTTP Methods:
+ * ACBS durable functions API library deals with following HTTP methods:
+ *
  * 1. GET
  * 2. POST
  * 3. PUT
  * 4. PATCH
  *
- * All the function have argument validation check and return object verification in
+ * All of the function have argument validation check and return object verification in
  * case err object does not have expected properties due to network connection, SSL verification or other issues.
  */
-const acbs = process.env.MULESOFT_API_UKEF_TF_EA_URL;
-const username = process.env.MULESOFT_API_KEY;
-const password = process.env.MULESOFT_API_SECRET;
-const axios = require('axios');
 
 require('dotenv').config();
+const axios = require('axios');
+
+// Headers declaration
+const tfs = {
+  [String(process.env.APIM_TFS_KEY)]: process.env.APIM_TFS_VALUE,
+  'Content-Type': 'application/json',
```

Comment:
can we check if we actually need this header? I assumed that by default the content-type is application/json